### PR TITLE
PLAT-1167 | Stop making requests that we know will end up with 403

### PR DESCRIPTION
**Pull request by rogatty**
https://bethink.atlassian.net/browse/PLAT-1167

### CODE REVIEW

File: resources/assets/js/router.js [hunk 1]
```diff
@@ -39,16 +39,17 @@ import ModeratorsDashboard from 'js/components/moderators/ModeratorsDashboard';
 import MainUsers from 'js/components/users/MainUsers';
 import UserProfile from 'js/components/users/UserProfile';
 import Onboarding from 'js/components/onboarding/Onboarding';
+import SplashScreen from 'js/components/global/splashscreens/SplashScreen';
 
 Vue.use(Router);
 
 const routes = [
 	{
-		name: 'course',
```
Comment (kvas-damian) [inline resources/assets/js/router.js]: what was wrong with this `name`?

**Comment by kvas-damian** [comment]
Navigation is missing on the splash screen page.

File: resources/assets/js/router.js (hunk 1 above)
Comment (rogatty) [reply resources/assets/js/router.js]: It was causing a warning:
```
[vue-router] Named Route 'course' has a default child route. When navigating to this named route (:to="{name: 'course'"), the default child route will not be rendered. Remove the name from this route and use the name of the default child route for named links instead.
```
I checked that we don't navigate to `'course'` route anywhere.

**Comment by rogatty** [comment]
>Navigation is missing on the splash screen page.
Fixed in https://github.com/bethinkpl/wnl-platform/pull/1429/commits/d974b8ab48f5203199ecac8ec4985625469ab72f

**Pull request merged**